src/main-process/parse-command-line.js

```diff
@@ -148,11 +154,6 @@ module.exports = function parseCommandLine(processArgs) {
     };
   }
 
-  if (args.help) {
-    process.stdout.write(options.help());
-    process.exit(0);
-  }
-
```

Comment:
Does yargs handle exiting when  `--help` is passed?

Reply:
I'm a little hazy on the details since I authored this, but I recall that yes it does. See the comment added on L145: "if --help or --version are given, this also displays the relevant message and exits"

Reply:
Great! Thanks for the explanation.